docs/preview.md

```diff
@@ -63,6 +77,13 @@ config file:
     extend-select = ["HYP001"]
     ```
 
+=== "CLI"
+
+    ```console
+    ruff check . --select HYP001
```

Comment:
The `.` is superfluous in all of these examples, that's the default behavior these days.

Reply:
We're using `extend-select` in all of the other examples, why use `--select` here?

Reply:
> The . is superfluous in all of these examples, that's the default behavior these days.

I chose to use the `.` to be consistent with the rest of the documentation (for example, see [the tutorial page](https://docs.astral.sh/ruff/tutorial/)). If it's better to remove the `.`, then I can open another PR to remove it from the rest of the documentation, what do you think ?

> We're using extend-select in all of the other examples, why use --select here?

It was a mistake, I fixed it in 9f5d9bcdf.

Reply:
Sure we can address that separately, thanks!

Reply:
Ok, I'll do that then. I've removed the `.` in this commit [356c4b0](https://github.com/astral-sh/ruff/pull/10168/commits/356c4b0580f63a918936a04b061b6503a28470d0).